lib/Core/Executor.cpp

```diff
@@ -4343,7 +4343,7 @@ void Executor::executeMemoryOperation(
           ObjectState *wos = bound->addressSpace.getWriteable(mo, os);
           wos->write(mo->getOffsetExpr(address), value);
           state.symbolicError->executeStore(address, value, error,
-                                            valueWithError);
+                                            valueWithError, target);
```

Comment:
@Himeshi If the last argument's type is changed to `llvm::Instruction *`, then `target` must change to `target->inst`.